backend/src/main/java/com/bakdata/conquery/models/config/ColumnConfig.java

```diff
@@ -77,40 +77,34 @@
 	 *
 	 * Also Name of output column for {@link FrontendConfig.UploadConfig#getIdResultInfos()}, ergo output csv-Columns.
 	 */
-	@InternalOnly
 	private String field;
```

Comment:
@awildturtok Weißt du warum hier `@InternalOnly` dranne war? Das Problem war hier das die Werte beim kopieren der Config in Standalone nicht mitgenommen wurden